ironic-python-agent/0001-Bumping-version-of-debian-to-stretch-adding-user-cor.patch

```diff
@@ -0,0 +1,124 @@
+-    proxy.sh apt-get --only-upgrade -t jessie-backports install -y qemu-utils
++    proxy.sh apt-get install -y --no-install-recommends gdisk python2.7 python2.7-dev python2.7-setuptools lvm2 \
++        python-pip qemu-utils parted hdparm util-linux genisoimage git gcc udev mdadm wget unzip alien libguestfs-tools \
++        bash coreutils tgt dmidecode ipmitool psmisc dosfstools pciutils && \
++    proxy.sh apt-get install -y linux-image-amd64 && \
++    proxy.sh apt-get --only-upgrade -t stretch-backports install -y qemu-utils
+ 
+ # Some cleanup
+ RUN proxy.sh apt-get -y autoremove && \
+@@ -44,6 +49,9 @@ RUN proxy.sh pip install -c /tmp/ironic-python-agent/upper-constraints.txt --no-
+ # This will succeed because all the dependencies were installed previously
+ RUN proxy.sh pip install -c /tmp/ironic-python-agent/upper-constraints.txt --no-cache-dir /tmp/ironic-python-agent
+ 
++# Install megacli
++RUN /tmp/get_megacli.sh
++
+ # Remove no longer needed packages
+ # NOTE(jroll) leave git to avoid strange apt issues in downstream Dockerfiles
+ # that may inherit from this one.
+diff --git a/get_megacli.sh b/get_megacli.sh
+new file mode 100755
+index 0000000..d296914
+--- /dev/null
++++ b/get_megacli.sh
+@@ -0,0 +1,16 @@
++#!/usr/bin/env sh
++
++VERSION=${VERSION:-'8-07-14'}
++# replace - with .
++RPM_VERSION=${VERSION//\-/\.}
++# get MegaCli
++wget https://docs.broadcom.com/docs-and-downloads/raid-controllers/raid-controllers-common-files/${VERSION}_MegaCLI.zip
++# extract it
++unzip ${VERSION}_MegaCLI.zip -d megacli
++cd megacli/Linux
++#change to deb and install
++alien -k MegaCli-${RPM_VERSION}-1.noarch.rpm
++dpkg -i megacli_${RPM_VERSION}-1_all.deb
++# clean
++rm -rf $(dirname "$0")/${VERSION}_MegaCLI.zip
++rm -rf $(dirname "$0")/megacli/
+diff --git a/imagebuild/coreos/coreos-oem-inject.py b/imagebuild/coreos/coreos-oem-inject.py
+index 0c7a92e..a5ebb6a 100755
+--- a/imagebuild/coreos/coreos-oem-inject.py
++++ b/imagebuild/coreos/coreos-oem-inject.py
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env python
++#!/usr/bin/env python2
+ 
+ #    Licensed under the Apache License, Version 2.0 (the "License"); you may
+ #    not use this file except in compliance with the License. You may obtain
+@@ -29,6 +29,7 @@ def get_coreos_version(version_file, key='COREOS_VERSION_ID'):
+                 return line.strip().split("=")[1]
+     raise ValueError('Could not read CoreOS version number from file')
+ 
++
+ COREOS_VERSION_FILE = os.path.join(os.path.dirname(__file__), 'version.txt')
+ COREOS_VERSION = os.environ.get('COREOS_VERSION',
+                                 get_coreos_version(COREOS_VERSION_FILE))
+diff --git a/imagebuild/coreos/oem/cloud-config.yml b/imagebuild/coreos/oem/cloud-config.yml
+index 03f71ce..33234e2 100644
+--- a/imagebuild/coreos/oem/cloud-config.yml
++++ b/imagebuild/coreos/oem/cloud-config.yml
+@@ -1,5 +1,10 @@
+ #cloud-config
+ 
++users:
++  - name: "core"
++    groups:
++      - "sudo"
```

Comment:
The core user used to implicitly have sudo, is that no longer true?

Reply:
This was for debugging purposes, I will delete it